### DISCUSSION TITLE
게임잼, 해커톤 관련 질문

### DISCUSSION THREAD

**Discussion by fkdl0048** [fkdl0048/Mentoring-Project#93] [Q&A] [answered]
멘토님은 제한된 시간내에 목표를 개발해야하는 과정에서 설계 레벨에서 빠르게 쳐내야 한다면 주먹구구식으로 만들어도 될까요?

그럼에도 구조를 생각하면서 만드는게 도움이 될까요?

**Comment by jongfeel** [comment, marked as answer]
음 복잡한 질문이군요.

설계를 해야 한다 말아야 한다가 중요하다기 보다는
팀에서 중요하다고 생각한 걸 정하고 빠르게 하고 있느냐를 보는게 더 나을 것 같습니다.
그 안에서 설계가 있으면 좋다고 생각하지만, 설계를 포기할 정도로 중요한 게 있었다면 그 결과를 반드시 확인해 보면 좋습니다.
